packages/backend/src/ee/database/migrations/20250516163613_migrate_slack_channel_project_mappings_table.ts

```diff
@@ -0,0 +1,63 @@
+// Migration 05: Migrate data from slack_channel_project_mappings to agents
+import { Knex } from 'knex';
+
+interface SlackMapping {
+    project_uuid: string;
+    organization_uuid: string;
+    slack_channel_id: string;
+    available_tags: string[] | null;
+}
+
+export async function up(knex: Knex): Promise<void> {
+    if (!(await knex.schema.hasTable('slack_channel_project_mappings'))) return;
+
+    const slackMappings = await knex('slack_channel_project_mappings').select(
+        'slack_channel_project_mapping_uuid',
+        'project_uuid',
+        'organization_uuid',
+        'slack_channel_id',
+        'available_tags',
+    );
+
+    // Process each mapping and create agents
+    // Sequential is fine because there're very few rows
+    for await (const mapping of slackMappings) {
+        try {
+            const [agent] = await knex('ai_agent')
+                .insert({
+                    project_uuid: mapping.project_uuid,
+                    organization_uuid: mapping.organization_uuid,
+                    name: `Legacy Agent`,
```

Comment:
I wouldn't call this `Legacy Agent` IMO. Users can build on top of this migrated content and start working on it.
Can you have multiple agents with the same name? So just "Agent"?

And description could be like 
`Agent for channel {channel name} for project {project name}` wdyt?